tags/cards/event.html

```diff
@@ -14,29 +14,29 @@
     				</div>
     			</div>
     		</div>
-            <div class="events" layout="row" layout-align="space-between">
-    			<div
-    				layout="column"
-    				class="day"
-    				each={day, rooms in schedule}>
-    				<h2>{day}</h2>
-    				<div layout="column" each={room, events in rooms}>
-    					<h3>{room}</h3>
-    					<div
-                            onclick={showDetail}
-    						class="event"
-    						each={time, event in events}
-    						style="
-    							background: hsl({event.hue}, 50%, 50%);
-    							top: {getTimeSpan(time).offset*scale}em;
-    							height: {getTimeSpan(time).height*scale}em"
-    						>
-    						<h4>{event.title}</h4>
-    						{event.description}
-    					</div>
-    				</div>
-    			</div>
-    		</div>
+                <div class="events" layout="row" layout-align="space-between">
```

Comment:
There are problems with the identation @Kato-San . Please have a look and push again :)